### PR TITLE
User can update a note by id.

### PR DESCRIPTION
**Pull request by riley-rangel**
api-issue-3 0

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -57,4 +57,23 @@ app.get('/notes', (req, res) => {
   })
 })
 
+app.put('/note-id/:id', (req, res) => {
+  const id = req.params.id
+  MongoClient.connect('mongodb://localhost/notepad', (error, db) => {
+    if (error) {
+      console.error(error)
+      res.sendStatus(404)
```
Comment (thebearingedge) [inline index.js]: `404` means "Not Found".

File: index.js [hunk 2]
```diff
@@ -57,4 +57,23 @@ app.get('/notes', (req, res) => {
...
+      process.exit(1)
+    }
+    const notes = db.collection('notes')
+    notes.updateOne({'_id': id}, {$set: {'content': req.body.message}})
+      .then(() => {
+        return notes.find({'_id': id}).toArray()
```
Comment (thebearingedge) [inline index.js]: Try `findOne`.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Great, almost there!

File: index.js (hunk 2 above)
Comment (riley-rangel) [reply index.js]: Agreed, much better. Thank you!

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**